### DISCUSSION TITLE
申请退款：9月 15 日反馈lantern无法登录专业版账号的问题，至今未解决

### DISCUSSION THREAD

**Discussion by dongc** [getlantern/lantern-forum-cn#935] [General]
申请退款：9月 15 日反馈lantern 登录上会员的问题，至今未解决

**Comment by dongc** [comment]
反馈问题：
https://github.com/getlantern/lantern-forum-cn/discussions/866

**Reply by dongc** [reply]
@Mike777Khoo 申请退款

**Reply by dongc** [reply]
dongc2010@gmail.com

**Comment by MaxMKMax** [comment]
@dongc 您好. 请您提供我支付单据. 我帮您申请退款.

**Reply by dongc** [reply]
![368012050](https://github.com/user-attachments/assets/f4f3b9de-d7ae-4c1c-8693-59e1e6e52036)

**Reply by dongc** [reply]
@Mike777Khoo 请问什么时候退款，退款到什么地方。
